packages/page-accounts/test/pages/accountsPage.tsx

```diff
@@ -30,59 +33,11 @@ class NotYetRendered extends Error {
 }
 
 // utility wrapper over an account item in accounts table, serves basic assertions about an account row
```

Comment:
This comment belongs with the extracted `AccountRow`.

Reply:
✅